Makefile

```diff
@@ -462,21 +463,22 @@ test-and-cover:
 		tail -n +2 profile_tmp.cov >> profile.cov; \
 	done
 
-android-lib: docker-golang-android genconfig
+android-lib-dev: docker-mobile
```

Comment:
Yeah, sounds good. Maybe `android-lib` for development and `android-lib-dist` for distribution?